esx_service/utils/auth_api.py

```diff
@@ -299,8 +299,20 @@ def _tenant_create(name, description="", vm_list=None, privileges=None):
     if not description:
         description = ""
 
-    logging.debug("_tenant_create: vms=%s", vms)
+    error_info, tenant_list = get_tenant_list_from_db()
```

Comment:
This comment is not for the code change, but for the code structure itself. 
tenant_create() is really 2 steps  (1) create empty tenant (2) add VMs if needed. 
so it should simply call _tenant_vm_add instead of creating another path of doing the same. 
//CC @lipingxue